examples/nsmix-mpi.py

```diff
@@ -56,7 +56,7 @@
 from mirgecom.steppers import advance_state
 from mirgecom.boundary import (  # noqa
     AdiabaticSlipBoundary,
-    IsothermalNoSlip,
+    IsothermalNoSlipBoundary,
```

Comment:
Yeah, I had this before, but replaced it in favor of the version without the "Boundary" designation.  Do we really want to put "Boundary" on the end of every boundary condition?  It is OK by me if this is your preference, but I was sort of leaning the other way with it (i.e. removing the "Boundary" designation.)